server/src/test/java/org/apache/druid/client/coordinator/CoordinatorClientImplTest.java

```diff
@@ -428,7 +450,7 @@ public void test_getCompactionSnapshots_nonNullDataSource() throws Exception
         AutoCompactionSnapshot.builder("ds1").build()
     );
     serviceClient.expectAndRespond(
-        new RequestBuilder(HttpMethod.GET, "/druid/coordinator/v1/compaction/status?dataSource=ds1"),
+        new RequestBuilder(HttpMethod.GET, "/druid/coordinator/v1/compaction/status?datasources=ds1"),
```

Comment:
We shouldn't be changing an existing API or its tests.

Reply:
my bad.